action.php

```diff
@@ -107,14 +107,16 @@ public function handle_before(Doku_Event $event, $param) {
         $act = act_clean($event->data);
         if($act == 'logout') {
             $this->_log('logged off');
-        } elseif(!empty($_SERVER['REMOTE_USER']) && $act == 'login') {
-            if(isset($_REQUEST['r'])) {
-                $this->_log('logged in permanently');
+        } elseif($act == 'login') {
+            if(!empty($_SERVER['REMOTE_USER'])) {
+                if(isset($_REQUEST['r'])) {
+                    $this->_log('logged in permanently');
+                } else {
+                    $this->_log('logged in temporarily');
+                }
             } else {
-                $this->_log('logged in temporarily');
+                $this->_log('failed login attempt');
```

Comment:
Unless I'm reading this wrong, this would log a failed attempt every time someone opens the login page, even though they haven't even tried to login yet.

Reply:
To be honest, I really don't know, as I wrote this code 3 years ago and haven't revised it.

I saw that you have closed all issues in this plugin. Many thanks!
I'll take a closer look later.

Reply:
@splitbrain Confirmed your change, simpler and way better than mine. Thank you.